src/uct/ib/rc/base/rc_ep.h

```diff
@@ -202,16 +202,22 @@ struct uct_rc_ep {
     ucs_list_link_t     list;
     ucs_arbiter_group_t arb_group;
     uct_rc_fc_t         fc;
+    uint16_t            atomic_mr_offset;
     uint8_t             path_index;
 };
 
 UCS_CLASS_DECLARE(uct_rc_ep_t, uct_rc_iface_t*, uint32_t, const uct_ep_params_t*);
 
 
 typedef struct uct_rc_ep_address {
+    uint8_t          flags;
```

Comment:
why not just always pack atomic mr id (or 0), like in `rc_mlx5`?
the address size would be the same, but it will require less code

Reply:
we could indeed but this is infrastructure for more address fields. @yosefe?

Reply:
then maybe extend this approach for `uct_rc_mlx5_ep_address_t` as well?

Reply:
we move it later to an new base class "rc_ep_address_t", as part of keepalive task (for passing remote access address/key)